railgun/internal/proxy/clientgo_cached_proxy_resolver.go

```diff
@@ -216,18 +217,25 @@ func (p *clientgoCachedProxyResolver) startCacheServer() {
 				p.logger.Error(err, "object could not be updated in the cache and will be discarded")
 				break
 			}
+			p.backendNeedsSync = true
 		case cobj := <-p.del:
 			if err := p.cacheDelete(cobj); err != nil {
 				p.logger.Error(err, "object could not be deleted from the cache and will be discarded")
 				break
 			}
+			p.backendNeedsSync = true
```

Comment:
Skip the configure sync optimization (https://github.com/Kong/kubernetes-ingress-controller/blob/next/pkg/sendconfig/sendconfig.go#L43)
will bring a lot of repeated kongUpdater call, please do not do that.

Reply:
I think this would _override_ reverse sync, though in a way that reduces calls, not increase them. We probably just need to update the condition later to `if !p.backendNeedsSync && !p.enableReverseSync` to continue forcing sync when reverse sync is enabled.

Reply:
> Skip the configure sync optimization (https://github.com/Kong/kubernetes-ingress-controller/blob/next/pkg/sendconfig/sendconfig.go#L43) will bring a lot of repeated kongUpdater call, please do not do that.
> I think this would override reverse sync, though in a way that reduces calls, not increase them. We probably just need to update the condition later to if !p.backendNeedsSync && !p.enableReverseSync to continue forcing sync when reverse sync is enabled.

Good call, accounted for now in 2ec390d4a5b100ee1c75042d517672cf4d0f61dd thanks team! :vulcan_salute: